<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditPointDialog</class>
 <widget class="QDialog" name="EditPointDialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>249</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>249</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>2</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="latLabel">
         <property name="text">
          <string>Ш:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="latSpinBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="decimals">
          <number>5</number>
         </property>
         <property name="minimum">
          <double>-90.000000000000000</double>
         </property>
         <property name="maximum">
          <double>90.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="lonLabel">
         <property name="text">
          <string>Д:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="lonSpinBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="decimals">
          <number>5</number>
         </property>
         <property name="minimum">
          <double>-180.000000000000000</double>
         </property>
         <property name="maximum">
          <double>180.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <widget class="QLabel" name="nameLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Наименование</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="nameLineEdit">
       <property name="maxLength">
        <number>128</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QComboBox" name="typeComboBox">
       <item>
        <property name="text">
         <string>----</string>
        </property>
        <property name="icon">
         <iconset resource="../images.qrc">
          <normaloff>:/gui/icons/pg_icons/qu.png</normaloff>:/gui/icons/pg_icons/qu.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Ограничение скорости</string>
        </property>
        <property name="icon">
         <iconset resource="../images.qrc">
          <normaloff>:/gui/icons/pg_icons/50.png</normaloff>:/gui/icons/pg_icons/50.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Внимание, дети!</string>
        </property>
        <property name="icon">
         <iconset resource="../images.qrc">
          <normaloff>:/gui/icons/pg_icons/children.png</normaloff>:/gui/icons/pg_icons/children.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Прочие опасности</string>
        </property>
        <property name="icon">
         <iconset resource="../images.qrc">
          <normaloff>:/gui/icons/pg_icons/att.png</normaloff>:/gui/icons/pg_icons/att.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Лежачий полицейский</string>
        </property>
        <property name="icon">
         <iconset resource="../images.qrc">
          <normaloff>:/gui/icons/pg_icons/sb.png</normaloff>:/gui/icons/pg_icons/sb.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Дикие животные</string>
        </property>
        <property name="icon">
         <iconset resource="../images.qrc">
          <normaloff>:/gui/icons/pg_icons/animals.png</normaloff>:/gui/icons/pg_icons/animals.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Железнодорожный переезд</string>
        </property>
        <property name="icon">
         <iconset resource="../images.qrc">
          <normaloff>:/gui/icons/pg_icons/rc.png</normaloff>:/gui/icons/pg_icons/rc.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ДПС</string>
        </property>
        <property name="icon">
         <iconset resource="../images.qrc">
          <normaloff>:/gui/icons/pg_icons/dps.png</normaloff>:/gui/icons/pg_icons/dps.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Камера контроля скорости</string>
        </property>
        <property name="icon">
         <iconset resource="../images.qrc">
          <normaloff>:/gui/icons/pg_icons/camera.png</normaloff>:/gui/icons/pg_icons/camera.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Опасный перекресток</string>
        </property>
        <property name="icon">
         <iconset resource="../images.qrc">
          <normaloff>:/gui/icons/pg_icons/att.png</normaloff>:/gui/icons/pg_icons/att.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Опасный поворот</string>
        </property>
        <property name="icon">
         <iconset resource="../images.qrc">
          <normaloff>:/gui/icons/pg_icons/att.png</normaloff>:/gui/icons/pg_icons/att.png</iconset>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Ограничение скорости:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="speedSpinBox">
         <property name="maximum">
          <number>99999</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>EditPointDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>EditPointDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
